src/helpers/UA/index.ts

```diff
@@ -442,7 +435,19 @@ export default class UAExtended extends UAConstructor implements UAExtendedInter
                 if (session) {
                     session.receiveRequest(request)
                 } else {
-                    logger.debug('received NOTIFY request for a non existent subscription')
+                    if (request.body) {
+                        const bodyParsed = JSON.parse(request.body) || {}
```

Comment:
Maybe wrap into try/catch? Parsing json from external source not always safest thing to do...